BAT/BCEngineMCMC.h

```diff
@@ -696,7 +696,7 @@ class BCEngineMCMC
 
     /**
      * Sets the number of Markov chains which are run in parallel. */
-    void SetNChains(unsigned n)
+    virtual void SetNChains(unsigned n)
```

Comment:
On further contemplation, issue #146 is junk. User-dependent initialization should be handled by `MCMCUserInitialization`, which is hopefully called in the right places. So this function should not be virtual (especially since it is indirectly called in the constructor).

Reply:
Reverted.